operator/config/crd/bases/forklift.konveyor.io_migrations.yaml

```diff
@@ -432,6 +432,9 @@ spec:
                     type:
                       description: Type used to qualify the name.
                       type: string
+                    vmFimware:
```

Comment:
there's a typo there (Fimware -> Firmware), I think just `firmware` would be alright